lessons/install-git-python-r.md

```diff
@@ -12,21 +12,57 @@ Installation instructions for Git can be found on the
 2. Anaconda also includes graphical interfaces for interacting with Python,
-   which can be invoked by running `spyder` or `jupyter-lab` from the command line.
-3. To install a new python package from the Anaconda repositories, simply run `conda install <package name>` in a terminal. You can also use the `pip` package manager, but it will be easier to keep track of packages by sticking to one installation method.
-4. To get an introduction on how to use Python, please read the tutorial over at [Software Carpentry](https://swcarpentry.github.io/python-novice-inflammation/).
-
-## User submitted packages
-
-Some packages are not available in the default Anaconda repositories. User contributed packaged are available in Anaconda "channels", use `anaconda search -t conda <package name>`, to find a channel with the desired package. To install this package, use `conda install -c <channel name> <package name>`. The [conda forge channel](https://conda-forge.github.io/) channel has many of the packages not in the default repositories, and the [bioconda](https://bioconda.github.io/) channel specifically targets bioconductor packages.
+   which can be invoked by running `spyder` or `jupyter-lab` from the command
+   line.
+3. To get an introduction on how to use Python, please read the tutorial over
+   at [Software
+   Carpentry](https://swcarpentry.github.io/python-novice-inflammation/).
+
+### Installing packages
+
+1. To install a new Python package from the Anaconda repositories, simply run
+   `conda install <package name>` in a terminal. You can also use the `pip`
+   package manager, but it will be easier to keep track of packages by sticking
+   to one installation method.
+2. Some packages are not available in the default Anaconda repositories. User
+   contributed packaged are available in Anaconda "channels", use `anaconda
+   search -t conda <package name>`, to find a channel with the desired package.
+   To install this package, use `conda install -c <channel name> <package
+   name>`. The [conda forge channel](https://conda-forge.github.io/) channel
+   has many of the packages not in the default repositories.
+
+## Trying Python without installing
+
+There are a few online services that provide solutions to run Python and Jupyter
+Notebooks without installing anything. These often come with some limitations
+but is a simple way to just try things out:
+
+-   https://notebooks.ai/ Optional to create account (but necessary for saving
+    progress). Most up to date version of `seaborn`.
+-   https://codeocean.com/ Account required. Sign up with institutional email
+    for extra storage and run time. Flexible but requires slight setup of
+    specifying which packages to install.
+-   https://colab.research.google.com/notebooks/welcome.ipynb Account required.
+    Google branded design.
+-   https://jupyter.org/try No account required. Not possible to save progress.
+    Lots of font-related warnings when plotting in our tests.
```

Comment:
Same here; also, what warnings?

Reply:
OK, never mind, I see the warnings with JupyterLab, but not the plain Python one.